### DISCUSSION TITLE
Data Science Discussion

### DISCUSSION THREAD

**Discussion by Aekansh-Ak** [snigi-gupta/AttendingUniversityAtBuffalo#14] [Q&A] [answered]
Hi,

As you mentioned you know some DS graduates, I just wanted to know their views on the course. Was the course tailored as per the industry's needs? 
How difficult it was for them to find a job and the average salary they scored after graduation.

**Comment by snigi-gupta** [comment]
@Aekansh-Ak,

So here is the thing, I don't know anything about DS. I was an MSCS (Masters, Computer Science) student. You can actually check out the course structure on [UB's Data Science Course website](http://engineering.buffalo.edu/home/academics/grad/programs/data-sciences.html#descriptions). 

You may reach out to them, but ***please do follow the basic guidelines when reaching out to them.***: [Guidelines](https://gist.github.com/snigi-gupta/90b2ba5fa4d98eb4f529c72e146e0fef)

[Kishore Ravisankar](https://www.linkedin.com/in/kishore-ravisankar/)
[Akshay Adlakha](https://www.linkedin.com/in/akshayadlakha/)
[Mira](https://www.linkedin.com/in/mira-moukheiber/)
[Thankam](https://www.linkedin.com/in/thankam-abish/)
[Dithya](https://www.linkedin.com/in/dithya-sridharan45/)
[Karun Prashar](https://www.linkedin.com/in/karunparashar/)

**Comment by falcon97** [comment, marked as answer]
Hey @Aekansh-Ak,
I feel the program has a solid foundation in teaching DS concepts as there are courses that teach you programming, and the math behind various DS algorithms. Compared to other degree programs out there, it may not be very flexible in terms of the electives offered. But for what it's worth, this program is competent enough to equip you with the required skills. This depends on the kind of role you are working in. At least for me, databases, programming and ML courses were useful in internships and my current job.

For the most part, getting a job entirely depends on you. Your previous experience, ability to network with people and a good resume are some factors you can control. The average salary depends on the job role and the location you work in. You can get an estimate by looking it up on Google.

Hope that helps!

**Reply by Aekansh-Ak** [reply]
Hi Kishore,

Thank you for your insight, this review certainly clarifies my doubt about the program.
